lumigator/infra/mzai/helm/lumigator/templates/_helpers.tpl

```diff
@@ -49,3 +49,14 @@ Selector labels
 app.kubernetes.io/name: {{ include "lumigator.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
+
+{{/*
+Postgresql password secret name
+*/}}
+{{- define "lumigator.postgresqlPasswordSecretName" -}}
+{{- if .Values.existingPostgresPasswordSecretName }}
```

Comment:
What about using default directly?

Reply:
Yes, but I mean, is it different from using the `default` function from Helm?

Reply:
Yeah, I could use `default` or `ternary` here